<?xml version="1.0"?>

<!--|
    Copyright (C) 2019 Learning Algorithms and Systems Laboratory, EPFL, Switzerland
    Authors:  Harshit Khurana
    email:    harshit.khurana@epfl.ch
    website:  lasa.epfl.ch

    This file is part of i_am_project EPFL (impact aware manipulation).
|-->

<launch>

  <node name="spawn_big_box" pkg="gazebo_ros" type="spawn_model" 
  args="-file $(find i_am_project)/object_models/big_box.urdf -x 0.6 -y 0.8 -z 0.1 -urdf -model table_model" />


  <node name="spawn_box" pkg="gazebo_ros" type="spawn_model" 
  args="-file $(find i_am_project)/object_models/box.urdf -x 0.6 -y 0.35 -z 0.33 -urdf -model box_model" />

  
  <!-- Here is the node that allows me to hit the object in the iiwa simulator -->
  <node pkg = "i_am_project" type = "momentum_hit"  name = "momentum_hit" output="screen"/>


</launch>

